ui/admin/app/controllers/scopes/scope/groups/index.js

```diff
@@ -23,7 +22,7 @@ export default class ScopesScopeGroupsIndexController extends Controller {
 
   queryParams = ['search', 'page', 'pageSize'];
 
-  @tracked search = '';
```

Comment:
I made all the initial `search` states to `undefined` instead of empty string because it shouldn't affect our initial search logic and to make sure that we don't add an extra debounce on an initial load as `this.retrieveData?.lastPerformed?.args?.[0].search !== params.search;` could return true as the param would be an empty string while the last performed task would be `undefined`.

I didn't think it was necessary to make the check against all falsy values such as `null`, `undefined`, `''` but let me know if you think otherwise.